frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/properties/coordsProperty/coordsProperty.component.tsx

```diff
@@ -77,17 +78,21 @@ export const CoordsProperty = ({ value, label, onChange, onBlur, required, error
 		}
 	};
 
+	const throttledSetSelectedPin = useCallback(throttle( // prevents spam clicking which caused a bug where the pin incorrectly appeared selected
+		(selected) => TicketsCardActionsDispatchers.setSelectedTicketPin(selected ? null : pinId), 1000, { leading: true, trailing: false },
+	), [pinId]);
+
```

Comment:
Quickly selecting one pin after the other, keeps the first pin big, but without the black lines 

https://github.com/3drepo/3drepo.io/assets/43547703/a7186d4c-d727-4172-b476-939c1ce54708

Reply:
@sanmont3drepo @The-Daniel, this has been fixed in https://github.com/3drepo/3drepounity/tree/ISSUE_494